biolink-model.yaml

```diff
@@ -941,10 +953,19 @@ slots:
     exact_mappings:
       - WIKIDATA:Q902638
 
+  has food component:
```

Comment:
Isn't there a predicate that can better capture an edge between a `food` and a `food component`?

Reply:
Again,  (cc: @cmungall) `had food component` is not a relationship edge. It's an  intrinsic characteristic of the `food` itself.  Similar situation as the `has phenotypic feature`.

Reply:
@deepakunni3 once again, I have converted `has food component` (as well as `has nutrient`) to predicates, for the same reasons as noted for the `has phenotypic feature` discussion above.

This realization suggests one more change in an unrelated part of the model: the clinical data. For those, we currently have (from the initial 'exposure/outcome/cohort' modelling), three node properties similar to the above:

- `has drug`
- `has device`
- `has procedure`

To continue to be consistent with the `has phenotypic feature` and `has food component` revisions, we should convert these three node properties to slot predicates as well. I will do this in this PR (since I've sync'd the code of the PR with the master containing the above three node properties).